website/docs/basics/blocks-patterns.md

```diff
@@ -23,3 +23,30 @@ This means the only difference between our variations and block patterns are:
 - Patterns appear in the editor inside the special tab for patterns.
 
 *We will soon provide WP-CLI command to automaticity create block patterns, boilerplate class, for you to use.*
+
+### Remove all core patterns
+
+To remove all core patterns add this code to you `src/Blocks/Blocks.php` class.
```

Comment:
Maybe add a paragraph about "why you'd want to do this" ("remove all core patterns):

_Why? If you don't use core blocks, core patterns won't work. Don't confuse your users and remove them (but feel free to add new patterns based on your own custom blocks!)_

Reply:
done